pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-managing-user-state.md

```diff
@@ -490,7 +490,7 @@ We recommend validating the authenticity of query parameter values by using the
 </ul>
 <p>To do this on your non-AMP page, include the following JavaScript, which will remove all query parameters from the URL:</p>
 <pre>
-var href = location.href.replace(/\?[^#]+/, '');
+var href = location.href.replace(/\?[^{{'[% raw %]'}}#]{{'{% endraw %}'}}+/, '');
```

Comment:
Those changes should not be part of this PR.

Reply:
Thanks!